Only in kyotocabinet-ruby-1.14.modify: Makefile
Common subdirectories: kyotocabinet-ruby-1.14/doc and kyotocabinet-ruby-1.14.modify/doc
Common subdirectories: kyotocabinet-ruby-1.14/example and kyotocabinet-ruby-1.14.modify/example
Only in kyotocabinet-ruby-1.14.modify: kyotocabinet.bundle
diff -u kyotocabinet-ruby-1.14/kyotocabinet.cc kyotocabinet-ruby-1.14.modify/kyotocabinet.cc
--- kyotocabinet-ruby-1.14/kyotocabinet.cc	2010-07-27 15:46:34.000000000 +0900
+++ kyotocabinet-ruby-1.14.modify/kyotocabinet.cc	2010-08-06 17:01:33.000000000 +0900
@@ -622,7 +622,7 @@
     }
     case T_FLOAT: {
       char kbuf[kc::NUMBUFSIZ*2];
-      size_t ksiz = std::snprintf(kbuf, sizeof(kbuf), "%f", (double)NUM2DBL(vobj));
+      size_t ksiz = snprintf(kbuf, sizeof(kbuf), "%f", (double)NUM2DBL(vobj));
       if (ksiz > sizeof(kbuf) - 1) {
         ksiz = sizeof(kbuf) - 1;
         kbuf[ksiz] = '\0';
Only in kyotocabinet-ruby-1.14.modify: kyotocabinet.o
Only in kyotocabinet-ruby-1.14.modify: mkmf.log
